client/src/Header/header.style.css

```diff
@@ -0,0 +1,41 @@
+.nav-bar {
+    list-style-type: none; 
+    margin: 30px;
+    overflow: hidden;
+    width: 100%;
+    z-index: 9999;
+}
+
+.nav-bar li {
+    float: left;
```

Comment:
Isn't it on the left side by default? Why did we need to use float left?

Reply:
It makes the navigation elements stay side by side

Reply:
let's avoid floats and absolute position for styling. there are other ways to position the tabs, right?

Reply:
+1 look into using flexbox for this.